scripts/xtensa-build-zephyr.py

```diff
@@ -615,13 +581,21 @@ def main():
 	else:
 		print("Building platforms: {}".format(" ".join(args.platforms)))
 
-	# we made two modes mutually exclusive with argparse
-	if args.west_path:
-		# check west_path input + create symlink if needed
-		run_point_mode()
-	if args.clone_mode:
+	west_root_dir, west_manifest_path = find_west_workspace()
+	if not west_root_dir:
+		execute_command(["west", "init", "-l", f"{SOF_TOP}"], cwd=west_top)
+		west_root_dir, west_manifest_path = find_west_workspace()
```

Comment:
We do not need to find west workspace here, we had just created one one line above.

Reply:
This is needed to correct the `print()`s below.